rate: 50
# output
outputTopic: /pwm
lpwm: 0
rpwm: 1
minPwm: 64
maxPwm: 255
minVelocity: 0.0
maxVelocity: 1.0
# input
inputTopic: /adc
input: 0
minReading: 293
maxReading: 790
minPos: -1.4929
maxPos: 1.4929
threshold: 8
average: 8
# control
trajectoryTopic: /arm_velocity_controller/follow_joint_trajectory/goal
joint: base